.github/actions/run-eden-test/action.yml

```diff
@@ -11,6 +11,9 @@ inputs:
   suite:
     required: true
     type: string
+  info_file:
```

Comment:
I wonder if it is needed to have this configurable. Just like for collected logs we can simply always store it under a hard-coded name (`eve-info.tar.gz`).

Reply:
done